Tasks/Xcode/task.json

```diff
@@ -67,10 +67,10 @@
         {
             "name": "xcWorkspacePath",
             "type": "filePath",
-            "label": "Workspace Path",
+            "label": "Workspace/Project Path",
             "defaultValue": "**/*.xcodeproj/*.xcworkspace",
             "required": false,
-            "helpMarkDown": "Optional relative path from repo root where xcworkspace folder exists. Scheme must also be specified if value provided. Omit if you intend to use the -target flag under Advanced Arguments."
+            "helpMarkDown": "Optional relative path from repo root where xcworkspace folder exists. Path to the *.xcodeproj can also be specified. Scheme must also be specified if value provided. Omit if you intend to use the -target flag under Advanced Arguments."
```

Comment:
I will update the help message. Also Scheme is no longer required on the task since some project types don't require scheme to be specified.